src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
```

Comment:
1. Error 1: The method named `sum` is uncommented and has a misleading name. It is not clear how it differs from the `add` method that already exists.